metric-publishers/emf-metric-publisher/src/main/java/software/amazon/awssdk/metrics/publishers/emf/internal/MetricEmfConverter.java

```diff
@@ -47,8 +50,11 @@
  */
 @SdkInternalApi
 public class MetricEmfConverter {
+    private static final int MAX_RECORD_SIZE = 100;
+    private static final int MAX_METRIC_NUM = 100;
```

Comment:
We could use more explanations on those numbers. This is one of the few places where comments are beneficial :)